addons/stock_account/models/product_category.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    property_valuation = fields.Selection([
+        ('manual_periodic', 'Periodic (manual)'),
+        ('real_time', 'Perpetual (automated)')],
+        string='Inventory Valuation',
+        required=True, copy=True,
```

Comment:
The value of 'copy' defaults to True for all fields except computed fields and one2many fields .